Diplomatic.Core/Classes/PDFGenerator.cs

```diff
@@ -42,12 +43,18 @@ public IDiploma Generate(Template template)
 
             gfx.DrawImage(originalPdf, new XRect(0, 0, page.Width, page.Height));
 
-            XFont font = new XFont("Comic Sans MS", 20, XFontStyle.Bold);
             foreach (var field in template.Fields)
             {
                 (int x, int y, int h, int w) = field;
+                var fontName = "Comic Sans MS";
+                var fontColor = "Black";
+                var fontSize = (int)(h * 0.8);
+
```

Comment:
I think we should add font information to the Field class. Not all templates will want black text on a white background, and a few of them might even want a different font than Comic Sans.